Sources/Starknet/Providers/StarknetProvider/StarknetProvider.swift

```diff
@@ -153,6 +153,15 @@ public class StarknetProvider: StarknetProviderProtocol {
         return result.transactionReceipt
     }
 
+    public func getPathfinderTransactionStatus(hash: Felt) async throws -> StarknetGatewayTransactionStatus {
```

Comment:
Maybe name it the same as JsonRpcMethod, pathfinderGetTransactionStatus?

Reply:
I'm not sure> tt sounds odd and usually when someone wants to use getters they start typing `obj.get...` but I don't mind `pathfinderGetTransactionStatus` either.

Reply:
Just a suggestion, it's fine the way it is though.